I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 200, in script
    return afterchat
  File "game/script.rpy", line 200, in <module>
    return afterchat
NameError: name 'afterchat' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 200, in script
    return afterchat
  File "C:\Users\Мария\Downloads\renpy-8.2.3-sdk\renpy\ast.py", line 1481, in execute
    renpy.store._return = renpy.python.py_eval(self.expression)
  File "C:\Users\Мария\Downloads\renpy-8.2.3-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Мария\Downloads\renpy-8.2.3-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 200, in <module>
    return afterchat
NameError: name 'afterchat' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
code tyan 1.0
Fri Mar 28 11:36:03 2025
